project.clj

```diff
@@ -1,11 +1,11 @@
-(defproject ring/ring-mock "0.4.0"
+(defproject ring/ring-mock "0.5.0"
   :description "A library for creating mock Ring request maps"
   :url "https://github.com/ring-clojure/ring-mock"
   :license {:name "The MIT License"
             :url "http://opensource.org/licenses/MIT"}
-  :dependencies [[org.clojure/clojure "1.7.0"]
-                 [cheshire "5.8.1"]
-                 [ring/ring-codec "1.1.1"]]
+  :dependencies [[org.clojure/clojure "1.12.0"]
```

Comment:
The Clojure version in the dependencies should denote the minimum compatible version.

Reply:
Reverted clojure version 1.7.0